packages/backend/src/database/migrations/20240409160248_add_dashboard_tab.ts

```diff
@@ -0,0 +1,129 @@
+import { file } from 'googleapis/build/src/apis/file';
+import { Knex } from 'knex';
+import { v4 as uuidv4 } from 'uuid';
+
+const tableName = 'dashboard_tabs';
+const dashboardTable = 'dashboards';
+const dashboardVersionTable = 'dashboard_versions';
+const DashboardTilesTableName = 'dashboard_tiles';
+const DashboardTileTabUuidColumnName = 'tab_uuid';
+
+type DbTabs = {
+    uuid: string;
+    name: string;
+    is_default: boolean;
+    dashboard_id: number;
+    dashboard_version_id: number;
+};
+
+export async function up(knex: Knex): Promise<void> {
+    // create dashboard tab table
+    if (!(await knex.schema.hasTable(tableName))) {
+        await knex.schema.createTable(tableName, (tableBuilder) => {
+            tableBuilder.string('name').notNullable();
+            tableBuilder.uuid('uuid').notNullable().defaultTo(uuidv4());
+            tableBuilder.boolean('is_default');
```

Comment:
This column doesn't seem to be used.

Reply:
I still don't think the `is_default` is providing meaningful value. 🤔 
- If we delete the default tab there is no logic to set a new default 
- The code defaults to the first tab if it can't find the default one

If we remove this column we can rely only on the first tab available as being the default.

Reply:
Based on the ticket requirements when a user deletes a tab we will delete all tiles inside it.